docs/framework/install/run-net-framework-1-1-apps.md

```diff
@@ -1,5 +1,6 @@
 ---
 title: "Run .NET Framework 1.1 apps on Windows 8, Windows 8.1, or Windows 10"
+description: "Describes how to accommodate apps that require the .NET Framework 1.1 which is no longer supported on many versions of the Windows operating system."
```

Comment:
Is this better? Is this correct?

```suggestion
description: "Describes how to accommodate apps that require the .NET Framework 1.1 which is no longer supported by Microsoft."
```

Reply:
I don't see any place in docs that says MS still supports 1.1 in any context. This is probably correct.

Reply:
@richlander would probably know.

Reply:
If @richlander could weigh in -

If we're not sure that some version of windows still runs 1.1, I think we should stay with my original wording.